.changeset/polite-squids-poke.md

```diff
@@ -0,0 +1,7 @@
+---
+"@thirdweb-dev/react": minor
+"@thirdweb-dev/react-core": patch
+"@thirdweb-dev/react-native": patch
+---
+
+ConnectWallet v3 updates
```

Comment:
Can update this changelog with the api changes and how they can be used? Like the new theme, new props, etc

Reply:
sure